conans/test/integration/toolchains/cmake/test_ninja.py

```diff
@@ -103,16 +103,66 @@ def test_locally_build_windows(self):
         client.run_command("App")
         self.assertIn("main: Release!", client.out)
         self.assertIn("main _M_X64 defined", client.out)
-        self.assertIn("main _MSC_VER19", client.out)
-        self.assertIn("main _MSVC_LANG2014", client.out)
 
-        client.run_command('{} && dumpbin /dependents /summary /directives "App.exe"'.format(vcvars))
-        self.assertIn("MSVCP140.dll", client.out)
-        self.assertIn("VCRUNTIME140.dll", client.out)
+        win_host = textwrap.dedent("""[settings]
```

Comment:
All of this is unnecessary? The new ``check_vs_runtime`` utility, will check the runtime of the executable, what is this code adding? Better just call ``check_vs_runtime()`` for the above executable